sklearn/mixture/gaussian_mixture.py

```diff
@@ -648,3 +686,32 @@ def aic(self, X):
             The greater the better.
         """
         return -2 * self.score(X) * X.shape[0] + 2 * self._n_parameters()
+
+    @property
+    def precisions_(self):
+        self._check_is_fitted()
+        if self.covariance_type is 'full':
+            return np.array([np.dot(prec, prec.T)
+                            for prec in self.precisions_cholesky_])
+        elif self.covariance_type is 'tied':
+            return np.dot(self.precisions_cholesky_,
+                          self.precisions_cholesky_.T)
+        else:
+            return self.precisions_cholesky_ ** 2
```

Comment:
Ok, I propose to precompute them in `_set_parameters` which is executed at the end of fit.
If you prefer I can add a specific abstracted function in `base.py` like `_compute_attributes`.